webpack.config.cjs

```diff
@@ -17,10 +17,13 @@ module.exports = {
   devtool: "source-map",
   resolve: {
     plugins: [new ResolveTypeScriptPlugin()],
+    alias: {
+      "crypto": "crypto-browserify"
```

Comment:
Webpack doesn't allow to use webcrypto somehow. See https://github.com/angular/angular-cli/issues/1548